File: PAMO-Kotlin/BMI_Calculator/app/src/main/java/com/example/bmi_calculator/Questions.java
package com.example.bmi_calculator;

public class Questions {

    public String mQuestions[] = {
            "Która z tych żywności jest przetworzona?",
            "Co jest owocem?",
            "Ile należy dziennie wypić wody?",
            "Jakie oznaczenie ma koronawirus?",
            "Jaki kraj jest największym na świecie producentem kakao?",
            "Co jest najmniej zdrowe?",
            "Ile gram tłuszczu ma w sobie marchewka?",
            "Który owoc jest najzdrowszy?",
            "Jest zupą o konsystencji gulaszu, na bazie wołowiny lub jesiotra, z dodatkiem migdałów i orzechów. Charakterystyczna dla kuchni gruzińskiej. Mowa o:",
            "Skąd pochodzi SŁYNNA sałatka grecka?"
    };

    private String mChoices[][] = {
            {"jajko", "batonik", "gruszka", "woda"},
            {"pomidor", "por", "cebula", "kalafior"},
            {"200ml", "15l", "1l", "1.5l"},
            {"COVID-19", "CV", "SARS", "SARS-CoV-2"},
            {"Polska", "Senegal", "Wybrzeże Kości Słoniowej", "Meksyk"},
            {"Frytki", "Chipsy", "Banan", "Marchewka"},
            {"50g", "0g", "15g", "8g"},
            {"Jabłko", "Truskawka", "Pomarańcza", "Kiwi"},
            {"Charczo", "Tołmie", "Chinkali", "Puri"},
            {"Egipt", "Cypr", "Turcja", "Grecja"}
    };

    private String mCorrectAnswers[] = {
            "batonik",
            "pomidor",
            "1.5l",
            "SARS-CoV-2",
            "Wybrzeże Kości Słoniowej",
            "Chipsy",
            "0g",
            "Kiwi",
            "Charczo",
            "Grecja"
    };

    public String getQuestion(int a){
        String question = mQuestions[a];
        return question;
    }

    public String getChoice1(int a){
        String choice = mChoices[a][0];
        return choice;
    };

    public String getChoice2(int a){
        String choice = mChoices[a][1];
        return choice;
    };

    public String getChoice3(int a){
        String choice = mChoices[a][2];
        return choice;
    };

    public String getChoice4(int a){
        String choice = mChoices[a][3];
        return choice;
    };

    public String getChoice5(int a){
        String choice = mChoices[a][4];
        return choice;
    };

    public String getChoice6(int a){
        String choice = mChoices[a][5];
        return choice;
    };

    public String getChoice7(int a){
        String choice = mChoices[a][6];
        return choice;
    };

    public String getChoice8(int a){
        String choice = mChoices[a][7];
        return choice;
    };

    public String getChoice9(int a){
        String choice = mChoices[a][8];
        return choice;
    };

    public String getChoice10(int a){
        String choice = mChoices[a][9];
        return choice;
    };

    public String getCorrectAnswer(int a) {
        String answer = mCorrectAnswers[a];
        return answer;
    }
}

File: PAMO-Kotlin/BMI_Kotlin/app/src/main/java/com/example/bmi_calculator/Questions.kt
package com.example.bmi_calculator

class Questions {
    var mQuestions = arrayOf(
            "Która z tych żywności jest przetworzona?",
            "Co jest owocem?",
            "Ile należy dziennie wypić wody?",
            "Jakie oznaczenie ma koronawirus?",
            "Jaki kraj jest największym na świecie producentem kakao?",
            "Co jest najmniej zdrowe?",
            "Ile gram tłuszczu ma w sobie marchewka?",
            "Który owoc jest najzdrowszy?",
            "Jest zupą o konsystencji gulaszu, na bazie wołowiny lub jesiotra, z dodatkiem migdałów i orzechów. Charakterystyczna dla kuchni gruzińskiej. Mowa o:",
            "Skąd pochodzi SŁYNNA sałatka grecka?"
    )
    private val mChoices = arrayOf(arrayOf("jajko", "batonik", "gruszka", "woda"), arrayOf("pomidor", "por", "cebula", "kalafior"), arrayOf("200ml", "15l", "1l", "1.5l"), arrayOf("COVID-19", "CV", "SARS", "SARS-CoV-2"), arrayOf("Polska", "Senegal", "Wybrzeże Kości Słoniowej", "Meksyk"), arrayOf("Frytki", "Chipsy", "Banan", "Marchewka"), arrayOf("50g", "0g", "15g", "8g"), arrayOf("Jabłko", "Truskawka", "Pomarańcza", "Kiwi"), arrayOf("Charczo", "Tołmie", "Chinkali", "Puri"), arrayOf("Egipt", "Cypr", "Turcja", "Grecja"))
    private val mCorrectAnswers = arrayOf(
            "batonik",
            "pomidor",
            "1.5l",
            "SARS-CoV-2",
            "Wybrzeże Kości Słoniowej",
            "Chipsy",
            "0g",
            "Kiwi",
            "Charczo",
            "Grecja"
    )

    fun getQuestion(a: Int): String {
        return mQuestions[a]
    }

    fun getChoice1(a: Int): String {
        return mChoices[a][0]
    }

    fun getChoice2(a: Int): String {
        return mChoices[a][1]
    }

    fun getChoice3(a: Int): String {
        return mChoices[a][2]
    }

    fun getChoice4(a: Int): String {
        return mChoices[a][3]
    }

    fun getChoice5(a: Int): String {
        return mChoices[a][4]
    }

    fun getChoice6(a: Int): String {
        return mChoices[a][5]
    }

    fun getChoice7(a: Int): String {
        return mChoices[a][6]
    }

    fun getChoice8(a: Int): String {
        return mChoices[a][7]
    }

    fun getChoice9(a: Int): String {
        return mChoices[a][8]
    }

    fun getChoice10(a: Int): String {
        return mChoices[a][9]
    }

    fun getCorrectAnswer(a: Int): String {
        return mCorrectAnswers[a]
    }
}